R/queue.R

```diff
@@ -25,9 +25,13 @@ Queue <- R6::R6Class("Queue", #nolint
                              "root is not version controlled."))
       }
 
+      # Connect to Redis
+      con <- redux::hiredis(host = redis_host())
+
       # Create queue
       self$controller <- rrq::rrq_controller(
-        queue_id %||% orderly_queue_id()
+        queue_id %||% orderly_queue_id(),
```

Comment:
Suggest erroring here or orderly_queue_id() here if not set?  but we could explore that in another PR if you prefer?

Reply:
i think we should error in the `main_worker` function because this is okay if we havent set the queue id, in fact we use that behaviour a lot in the tests, have made a ticket: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5663/orderly.runner-worker-error-if-ORDERLYRUNNERQUEUEID-not-set